lib/learn_to_rank/data_pipeline/embed_features.rb

```diff
@@ -58,11 +58,9 @@ def fetch_document(judgement)
           "fields" => %w(popularity content_id title format description link public_timestamp organisation_content_ids updated_at indexable_content),
           "count" => %w[20],
         }
-        @cached_queries[judgement[:query]] ||= do_fetch(query)
-        results = @cached_queries[judgement[:query]]
-        if @cached_queries.keys.count > 50
-          flush_cached_queries
-        end
+        cached_queries[judgement[:query]] ||= do_fetch(query)
+        results = cached_queries[judgement[:query]]
```

Comment:
Since `cached_queries` is already being treated like an instance variable here, rather than passing it to methods, should we keep it as `@cached_queries`?

Reply:
That makes sense.  As the judgements are grouped by queries, I'll replace this caching with two instance variables: `@last_search_term` and `@last_query`